lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -564,6 +572,37 @@ def fetch_ec2_metadata
       end
     end
 
+    # TODO: This functionality should eventually be available in another
+    # library, but implement it ourselves for now.
+    module CredentialsInfo
+      # Determine the project ID from the credentials, if possible.
+      # Returns the project ID (as a string) on success, or nil on failure.
+      def self.project_id
+        return nil if @auth_method == 'private_key'
+        creds = Google::Auth.get_application_default(LOGGING_SCOPE)
+        if creds.issuer
```

Comment:
No, it's "issuer" - http://www.rubydoc.info/github/google/signet/Signet/OAuth2/Client#issuer-instance_method